project1/templates/bookpage.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+
+<html class="no-js">
+
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">

```

Comment:
Use a generic template and extend it to this file.

Reply:
added layout.html and remaining files are extended.